templates/search.tmpl

```diff
@@ -5,56 +5,125 @@
                     Search: <input type="search">
                     <input type="submit">
                 </form>
-                <ul class="searchresults">
-                </ul>
+                    <div class="row">
+                      <div class="column left">
+                      <h3> Search on Keys </h3>
+                      <ul style="list-style-type:none">
+                        <li><input name="searchkey" type="checkbox" onclick="doSearch()" value="name" checked/> Name</li>
+                        <li><input name="searchkey" type="checkbox" onclick="doSearch()" value="login" checked/> Login</li>
+                        <li><input name="searchkey" type="checkbox" onclick="doSearch()" value="location" checked/> Location</li>
+                        <li><input name="searchkey" type="checkbox" onclick="doSearch()" value="email" checked/> Email</li>
+                      </ul>
+                      </div>
+                      <div class="column right">
+                        <div id="resultscount"></div>
+                        <table class="searchresults">
+                        </table>
+                      </div>
+                    </div>
 <style>
 .searchform input { color: black; }
 ul.searchresults {
     margin: 0;
     padding: 0;
 }
-ul.searchresults li:nth-child(1) {
+#resultscount {
     list-style: none;
     font-weight: bold;
     margin: 0;
     padding: 0;
 }
+a {
+    text-decoration: underline;
+}
+.column {
+    float: left;
+    padding: 10px;
+    box-sizing: border-box;
+}
+.left {
+  width: 20%;
+}
+.right {
+  width: 80%;
+}
+.row:after {
+    content: "";
+    display: table;
+    clear: both;
+    box-sizing: border-box;
+}
```

Comment:
The table styles need to work responsively; at the moment, on a narrow screen like a mobile, the table overflows the screen to the right which makes it really hard to read because you have to keep scrolling right and left. I don't know whether it's best to linearise the table or what here; perhaps it would be good if each row became two rows or similar.

Reply:
The table should also drop below the checkboxes rather than be to the right of them on narrow screens, which will help a lot too.